src/frontend/styles/components/sidebar.pcss

```diff
@@ -59,11 +59,45 @@
     }
   }
 
+  &__search {
```

Comment:
The color of the icon, text and a shortcut symbol should be the same. But now they are different

<img width="400" alt="image" src="https://user-images.githubusercontent.com/3684889/188509762-2bc43f50-4902-46d0-8b04-761730287b3d.png">

Reply:
I checked they all are of the same color only